tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator/BuildTasks/RemoveRuntimeDependencies.cs

```diff
@@ -32,8 +38,14 @@ public override bool Execute()
                 ignoreFilesSet.Add(item.ItemSpec);
             }
 
+            if (!_runtimeToAssemblyFileMap.TryGetValue(TargetFramework, out var assemblyListFileName))
+            {
+                Log.LogError($"The TargetFramework '{TargetFramework}' is not supported in this project.");
```

Comment:
Wondering if we could run into problems here. We might still have users targeting different TFMs other than net6.0 (e.g. .NET standard). We could still enforce this here, but wondering if we should improve the message to list the supported targets. Thoughts?